### PR TITLE
Task2 Calculator

### PR DESCRIPTION
**Pull request by IrynaSemenenko**
The program stops after entering an invalid value.

### CODE REVIEW

**Review by Hmeliumm — CHANGES_REQUESTED** [review]
Please pull master/main in this branch

**Review by Hmeliumm — CHANGES_REQUESTED** [review]
Please make it working (actually do arithmetic operatopns=)). Like, you entered a and b and operation "+". It should do a+b

File: Simple Calculator/src/Calculator.java [hunk 1]
```diff
@@ -0,0 +1,60 @@
...
+                    operation.equals("/") |
+                    operation.equals("%")) {
+                System.out.println("User inputs supported operations");
+
+                System.out.println("Enter the second number");
+                if (sc.hasNextDouble()) {
+                    double secondValue = sc.nextDouble();
+                    System.out.println("Input is correct: " + secondValue);
+                    if (operation.equals("+")) {
+                        double result = firstValue + secondValue;
```
Comment (Hmeliumm) [inline Simple Calculator/src/Calculator.java:27]: Please initialize variable at the beginning of method, and here and below just use it

File: Simple Calculator/src/Calculator.java [hunk 2]
```diff
@@ -0,0 +1,60 @@
...
+                        double result = firstValue * secondValue;
+                        System.out.println("Result: " + firstValue + operation + secondValue + "=" + result);
+                    } else if (operation.equals("/")) {
+                        if (firstValue == 0 | secondValue == 0) {
+                            System.out.println("Division by '0' is impossible. Try to select another operation or enter other data");
+                        } else {
+                            double result = firstValue / secondValue;
+                            System.out.println("Result: " + firstValue + operation + secondValue + "=" + result);
+                        }
+                    } else if (operation.equals("%")) {
```
Comment (Hmeliumm) [inline Simple Calculator/src/Calculator.java:42]: Could be just else

**Review by Hmeliumm — CHANGES_REQUESTED** [review]

File: Simple Calculator/src/Calculator.java (hunk 2 above)
Comment (IrynaSemenenko) [reply Simple Calculator/src/Calculator.java:43]: done

File: Simple Calculator/src/Calculator.java (hunk 1 above)
Comment (IrynaSemenenko) [reply Simple Calculator/src/Calculator.java]: done

**Review by Hmeliumm — APPROVED** [review]

**Review by Hmeliumm — APPROVED** [review]

**Pull request merged**